File: dataset_reduzido/codenet/Java/Code/s440824480.java
import java.util.*;
class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int c=0;
    int flg=0;
    while(n-->0){
      int x = sc.nextInt();
      int y = sc.nextInt();
      
      if(x==y){
        c++;
        if(c>=3){
        flg=1;
      }
      }
      else{
        c=0;
      }
    }
    if(flg==0){
      System.out.println("No");
    }
    else{
      System.out.println("Yes");
    }
  }
}

File: output/Llama-2-100/codenet/Java/Python/s440824480.py
def main():
    n = int(input())
    c = 0
    flg = 0
    while n > 0:
        x = int(input())
        y = int(input())
        if x == y:
            c += 1
            if c >= 3:
                flg = 1
        else:
            c = 0
    if flg == 0:
        print("No")
    else:
        print("Yes")
